README.md

```diff
@@ -13,7 +13,7 @@ In order to deploy the tracker, some boilerplate code is neccessary for configur
 1. [Install pip](https://pip.pypa.io/en/stable/installing/) This is the package management system we use with the tracker, and its generally the best option for getting Python packages.
 1. [Install node](https://nodejs.org/en/download/). Currently the JS code is only tested with node 12, but others may work.
 1. [Install yarn](https://yarnpkg.com/en/). If you have npm installed, you can just run `npm i -g yarn` and it should do the right thing.
-1. [Install direnv](https://github.com/direnv/direnv). *Optional, Linux/OSX only* This will help set up an isolated development environment. PyCharm might do the job on Windows, but no promises.
+1. [Install direnv](https://github.com/direnv/direnv). *Optional, Linux/OSX only* This will help set up an isolated development environment. PyCharm might do the job on Windows, but no promises. Be sure to add [the relevant code](https://github.com/direnv/direnv/wiki/Python#pyenv) to `.direnvrc`.
```

Comment:
nit: maybe "to _your_ `.direnvrc`"?

My understanding from the linked docs are that the file lives in the user's home directory, but reading this first I thought the file should exist in this repo's directory. Could just be me being unfamiliar, but then again that might be the kind of user reading these docs in the first place /shrug